CODEOWNERS

```diff
@@ -280,7 +280,9 @@
 /packages/authentication @emonddr @hacksparrow @jannyHou
 /packages/security @emonddr @hacksparrow @jannyHou
 /extensions/authentication-passport @emonddr @hacksparrow @jannyHou
+/extensions/authentication-jwt @emonddr @hacksparrow @jannyHou
 /docs/site/tutorials/authentication @emonddr @hacksparrow @jannyHou
+/examples/passport-login @deepakrkris @emonddr
```

Comment:
Similarly here, should this line use the same list of CODEOWNERS as the lines above?

Reply:
Hm. I'm not sure about this one. @deepakrkris created this example, so I assume he should be in the list? or are you suggesting the standby owners should be: 
```Standby owner(s): @emonddr @jannyHou @deepakrkris ```

Reply:
I don't mind either way. I just found it surprising that some lines in the same functional area have different owners than other lines.

Maybe we should use all FA owners (because they are FA owners) and add Deepak as the original author?

Reply:
I am good to be the standby owner :)